NICE.Identity.Authorisation.WebAPI/Controllers/UsersController.cs

```diff
@@ -429,5 +429,33 @@ public IActionResult GetUsersByOrganisationId(int organisationId)
                 return StatusCode(500, new ProblemDetails { Status = 500, Title = $"{e.Message}" });
             }
         }
+
+        /// <summary>
+        /// get users and job Ids by organisation id
+        /// </summary>
+        /// <param name="organisationId"></param>
+        /// <returns></returns>
+        [HttpGet("usersandjobIdsbyorganisation/{organisationId:int}")]
+        [ProducesResponseType(typeof(User), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [Produces("application/json")]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Policy = Policies.API.UserAdministration)]
+        public IActionResult GetUsersAndJobIdsByOrganisationId(int organisationId)
+        {
+            try
+            {
+                var user = _usersService.GetUsersAndJobIdsByOrganisationId(organisationId);
```

Comment:
Same comment as above - this 'user' variable looks to be a userandjobid object, not a user